packages/datadog_flutter_plugin/android/src/test/kotlin/com/datadoghq/flutter/DatadogSdkPluginTest.kt

```diff
@@ -659,4 +722,18 @@ class DatadogSdkPluginTest {
             assertThat(plugin.telemetryOverrides.trackFlutterPerformance).isEqualTo(trackFlutterPerformance)
         }
     }
+}
+
+@Suppress("EmptyCatchBlock")
+fun Any.getPrivate(varName: String): Any? {
+    var value: Any? = null
+    try {
+        val field = javaClass.getDeclaredField(varName)
+        field.isAccessible = true
+        value = field.get(this)
+    } catch (_: NoSuchFieldException) {
+
```

Comment:
maybe it is worth to log something?

Reply:
I figured since this was only for tests and `null` was essentially the failure state, it didn't matter, but I'll add one just in case.